website/docs/utilities/popover-primitive/partials/code/how-to-use.md

```diff
@@ -0,0 +1,150 @@
+  <:toggle>hover or focus me</:toggle>
+  <:content>your popover content here</:content>
+</Hds::PopoverPrimitive>
+```
+
+_Notice: from a pure technical standpoint, the events are `mouseEnter/Leave` and `focusIn/Out`._
+
+Alternatively, the toggle behaviour can be enabled via "click" events:
+
+```handlebars
+<Hds::PopoverPrimitive
+  @enableClickEvents={{true}}
+  class="doc-popover-primitive-custom-style"
+>
+  <:toggle>click me</:toggle>
+  <:content>your popover content here</:content>
+</Hds::PopoverPrimitive>
+```
+
+!!! Warning
+
+Important: if you don't apply any of the `@enableSoftEvents` or `@enableClickEvents` the popover will not become visible with any kind of interaction (unless it's rendered already opened via the special `@isOpen` argument).
+
+!!!
+
+### Content positioning
+
+The `:content` block can be positioned in relation to the `:toggle` block using the `placement` argument of the `@anchoredPositionOptions`:
+
+```handlebars
+<Hds::PopoverPrimitive
+  @anchoredPositionOptions={{(hash placement="top-start")}}
+  @enableSoftEvents={{true}}
+  class="doc-popover-primitive-custom-style"
+>
+  <:toggle>toggle</:toggle>
+  <:content>your popover content here</:content>
+</Hds::PopoverPrimitive>
+```
+
+### Content sizing
+
+The `:content` block can be assigned fixed horizontal/vertical sizes using the `@popoverWidth` and `@popoverHeight` arguments:
+
+```handlebars
+<Hds::PopoverPrimitive
+  @enableSoftEvents={{true}}
+  @popoverWidth="200px"
+  @popoverHeight="100px"
+  class="doc-popover-primitive-custom-style"
+>
+  <:toggle>toggle</:toggle>
+  <:content>your popover content here</:content>
+</Hds::PopoverPrimitive>
+```
+
+### Collision detection
+
+It's possible to enable the collision detection logic using the `enableCollisionDetection` argument of the `@anchoredPositionOptions`:
+
+```handlebars
+<Hds::PopoverPrimitive
+  @anchoredPositionOptions={{(hash enableCollisionDetection=true)}}
+  @enableSoftEvents={{true}}
+  class="doc-popover-primitive-custom-style"
+>
+  <:toggle>toggle</:toggle>
+  <:content>your popover content here</:content>
+</Hds::PopoverPrimitive>
+```
+
+For details about how the collision detection works refer to the [Floating UI > Tutorial](https://floating-ui.com/docs/tutorial).
+
+### Other `anchoredPositionOptions`
+
+There are more possible options and configurations that can be provided to the popover via the `@anchoredPositionOptions` argument. Refer to the [Component API](#component-api) section below for more details.
+
+### Applied to interactive elements
+
+If the popover is applied to an element that is interactive (e.g., a button, a link, an input) or that contains interactive elements as children, the "wrapping" `:toggle` element must be rendered as a generic `<div>` instead of a `<button>` to avoid nesting interactive elements (not accessible). To achieve that, you have to set `@toggleContainsInteractive` to `true` (in this case you have to use "soft" event listeners for the toggle):
```

Comment:
```suggestion
If the popover is applied to an element that is interactive (e.g., a button, a link, an input) or that contains interactive elements as children, the "wrapping" `:toggle` element must be rendered as a generic `<div>` instead of a `<button>` to avoid nesting interactive elements (not accessible). To achieve this, set `@toggleContainsInteractive` to `true` (in this case you have to use "soft" event listeners for the toggle):
```